concurrency/1.1/_index.md

```diff
@@ -0,0 +1,52 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Jakarta Concurrency 1.1"
+date: 2019-07-17T22:03:40+0100
+summary: "Jakarta EE 8 Release."
+---
+Jakarta Concurrency provides a specification for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
+
+* [Jakarta Concurrency 1.1 Specification Document](./concurrency_1.1.pdf) (PDF)
+* [Jakarta Concurrency 1.1 Specification Document](./concurrency_1.1.html) (HTML)
+* [Jakarta Concurrency 1.1 Javadoc](./apidocs)
+* [Jakarta Concurrency 1.1 TCK](http://downloads.eclipse.org/jakarta/concurrency/1.1/concurrency_tck_1.1.zip)
+* Maven coordinates
+  * [jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:jar:1.1.2](https://search.maven.org/artifact/jakarta.enterprise.concurrent/jakarta.enterprise.concurrent-api/1.1.2/jar)
+
+# Ballots
+
+## Release Review
+
+The Release Review Specification Committee Ballot concluded successfully on 2019-12-01 with the following results.
+
+|                       |  Yes    | No      | Abstain  |
+|-----------------------|---------|---------|----------|
+|Fujitsu                | &check; |         |          |
+|IBM                    | &check; |         |          |
+|Oracle                 | &check; |         |          |
+|Payara                 | &check; |         |          |
+|Red Hat                | &check; |         |          |
+|Tomitribe              | &check; |         |          |
+|EE4J PMC               | &check; |         |          |
+|Participant Members    |         | &check; |          |
+|Committer Members      | &check; |         |          |
+
+## Plan Review
+
+The Plan Review Specification Committee Ballot concluded successfully on 2019-09-31 with the following results.
+
+|                       |  Yes    | No  | Abstain  |
+|-----------------------|---------|-----|----------|
+|Fujitsu                | &check; |     |          |
+|IBM                    | &check; |     |          |
+|Oracle                 | &check; |     |          |
+|Payara                 | &check; |     |          |
+|Red Hat                | &check; |     |          |
+|Tomitribe              | &check; |     |          |
+|EE4J PMC               | &check; |     |          |
+|Participant Members    | &check; |     |          |
+|Committer Members      | &check; |     |          |
+
+# Compatible Implementations
+
+* Eclipse GlassFish 5.1
```

Comment:
Should include a link to the CI:  https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip

Reply:
Done